### ISSUE TITLE
read function for WorkMode

### ISSUE THREAD

**Issue by AlanGray** [unitreerobotics/unilidar_sdk2#2]
I've searched the SDK and didn't find a way to read WorkMode. If this doesn't exist, it should be added.

**Comment by CHartberger** [comment]
Correct. I would also like to check the actual settings beeing used by the lidar. Running the run.launch you can only see the loaded Parameters, but do they work?

PARAMETERS
 * /rosdistro: noetic
 * /rosversion: 1.17.4
 * /unitree_lidar_ros_node/baudrate: 4000000
 * /unitree_lidar_ros_node/cloud_frame: unilidar_lidar
 * /unitree_lidar_ros_node/cloud_scan_num: 18
 * /unitree_lidar_ros_node/cloud_topic: unilidar/cloud
 * /unitree_lidar_ros_node/imu_frame: unilidar_imu
 * /unitree_lidar_ros_node/imu_topic: unilidar/imu
 * /unitree_lidar_ros_node/initialize_type: 2
 * /unitree_lidar_ros_node/laserscan_frame: unilidar_laserscan
 * /unitree_lidar_ros_node/laserscan_topic: unilidar/laserscan
 * /unitree_lidar_ros_node/lidar_ip: 192.168.1.62
 * /unitree_lidar_ros_node/lidar_port: 6101
 * /unitree_lidar_ros_node/pc_ip: 192.168.1.2
 * /unitree_lidar_ros_node/pc_port: 6201
 * /unitree_lidar_ros_node/range_max: 100
 * /unitree_lidar_ros_node/range_min: 0
 * /unitree_lidar_ros_node/serial_port: /dev/ttyACM0
 * /unitree_lidar_ros_node/use_system_timestamp: True
 * /unitree_lidar_ros_node/work_mode: 1

I would like to switch to the wide-angle mode, but my analysis shows only roughly 92° azimuth angle instead of 96°.
